js/msp/MSPHelper.js

```diff
@@ -192,46 +193,63 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 break;
 
             case MSPCodes.MSP_VOLTAGE_METER_CONFIG:
-                VOLTAGE_METER_CONFIGS = [];
-                var voltage_meter_count = data.readU8();
-
-                for (var i = 0; i < voltage_meter_count; i++) {
-                    var subframe_length = data.readU8();
-                    if (subframe_length != 5) {
-                        for (var j = 0; j < subframe_length; j++) {
-                            data.readU8();
+                if (semver.lt(CONFIG.apiVersion, "1.36.0")) {
```

Comment:
Didn't think of that. But maybe Cleanflight users also want to use the GUI to change battery and current settings on older fw versions. @hydra can you look at this, we want to push a new version of the BFC to the chome store as soon as possible to fix all the incompatibility issues with BF 3.1.x